lib/localization/locales/tr.json

```diff
@@ -0,0 +1,63 @@
+{
+  "loading": "Yükleniyor...",
+  "why_am_i_seeing": "Bunu neden görüyorum?",
+  "protected_by": "Koruma sağlayan:",
+  "made_with": "🇨🇦'da ❤️ ile yapıldı",
+  "mascot_design": "Maskot tasarımı:",
+  "ai_companies_explanation": "Bunu görüyorsunuz çünkü bu web sitesinin yöneticisi, yapay zeka şirketlerinin web sitelerini agresif şekilde kazımasına karşı sunucuyu korumak için Anubis'i kurdu. Bu tarz kazımalar sitelerin erişilemez olmasına ve kesintilere neden olabiliyor.",
+  "anubis_compromise": "Anubis bir uzlaşmadır. Anubis, spam e-postaları azaltmak için önerilen bir iş kanıtı sistemi olan Hashcash benzeri bir sistemi kullanır. Bireysel kullanımda bu ek yük göz ardı edilebilir olsa da, büyük ölçekli kazıyıcılarda birikerek kazımayı oldukça maliyetli hale getirir.",
+  "hack_purpose": "Bu geçici bir çözümdür. Esas amacı, başsız tarayıcıları parmak iziyle tanımlamak için daha fazla zaman kazandırmak, ve bu sayede meşru kullanıcıların bu zorluk sayfasını görmesini önlemektir.",
```

Comment:
i think "Bu geçici çözümün esas amacı, başsız tarayıcıları parmak iziyle tanımlamak için mümkün olduğu kadar zaman kazandırmak, ve bu sayede meşru kullanıcıların bu zorluk sayfasını görmesini önlemektir." flows better here